Assets/CustomIK/Scripts/CustomIK.cs

```diff
@@ -0,0 +1,137 @@
+﻿using UnityEngine;

```

Comment:
Add `.DS_Store` files to gitignore and reset your git cache https://devconnected.com/how-to-clear-git-cache/

Reply:
put this comment here because I couldn't put it above